core/src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -228,11 +226,7 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
     //check that the owner is a part of both the access group and the intersection group
     Await.result(retry(when500)(() => Future { directory.members.get(ownerAG.groupEmail.value, gcsDAO.toProxyFromUser(testCreator)).execute() }), Duration.Inf)
     Await.result(retry(when500)(() => Future { directory.members.get(ownerIG.groupEmail.value, gcsDAO.toProxyFromUser(testCreator)).execute() }), Duration.Inf)
-
-    val projectOwnerAGMembers = Await.result(retry(when500)(() => Future { directory.members().list(projectOwnerAG.groupEmail.value).execute() }), Duration.Inf)
-    projectOwnerAGMembers.getMembers.map(_.getEmail) should be { Seq(projectOwnerGoogleGroup.groupEmail.value) }
-    val projectOwnerIGMembers = Await.result(retry(when500)(() => Future { directory.members().list(projectOwnerIG.groupEmail.value).execute() }), Duration.Inf)
-    projectOwnerIGMembers.getMembers.map(_.getEmail) should be { Seq(projectOwnerGoogleGroup.groupEmail.value) }
+    Await.result(retry(when500)(() => Future { directory.members.get(projectOwnerIG.groupEmail.value, gcsDAO.toProxyFromUser(testCreator)).execute() }), Duration.Inf)
```

Comment:
In this case, you kept the Google call but removed the `should`. Was that intentional or is there something that could be checked about the result?